src/scripts/sync_synonym.py

```diff
@@ -68,14 +68,14 @@ def _query_synonyms(ids: List[CURIE], db: SqlImplementation) -> pd.DataFrame:
 
 
 # todo: DRYify? this and _merge_synonym_types() share similar logic
-def _add_synonym_type_inferences(row: pd.Series) -> str:
+def _add_synonym_type_inferences(row: pd.Series, exclusions: Set[str]) -> str:
     """Append additional synonym_type(s) based on logical rules."""
     syn: str = row['synonym']
     syn_type_str: str = row['synonym_type']
     types: Set[URI] = set(syn_type_str.split('|') if syn_type_str else [])
 
     # Acronym: uppercase, no numbers, no whitespace, <10 chars
-    if syn.isupper() and not any(char.isspace() for char in syn) and len(syn) < 10:
+    if syn not in exclusions and syn.isupper() and not any(char.isspace() for char in syn) and len(syn) < 10:
```

Comment:
> Did you confirm from the curated examples that the list of synonyms that should not be tagged as an abbreviation are not tagged now?

Yep! I checked a few cases in the new [v8 google sheet](https://docs.google.com/spreadsheets/d/1ftEZ47-JDe-1Z_fRWyO5NS39qT_-8MZsHXHEKz9aZ9Q/edit?gid=978570179#gid=978570179), and they're longer being tagged as such.

Reply:
Thanks for the link to the data file. I checked all the ones in the [curated file](https://docs.google.com/spreadsheets/d/1y5jCmHN51ydBOcdqmGTBHjDeOHTPuYWd165qEYnNtIk/edit?gid=0#gid=0) and none are tagged as abbreviations now as expected with this PR.